src/send-conf-email.ts

```diff
@@ -128,6 +128,39 @@ export const handler = async (
     return input;
   } catch (err) {
     console.error(`Error sending email for event`, err);
```

Comment:
```suggestion
```
Looking good! Could you also remove this line so we're never logging the raw error message unless we're at the last branch in `notifyErrorHandler`?